### DISCUSSION TITLE
手機上可以玩GTA了！

### DISCUSSION THREAD

**Discussion by BeaGle0928** [1101-GS4719-C/Python-Community-GS4719-C#1733] [General]
Netflix 宣布，Rockstar Games旗下《俠盜獵車手：三部曲 最終版》（Grand Theft Auto：The Trilogy – The Definitive Edition）將於 2023 年 12 月 14 日起，在 Netflix Games 推出 iOS/Android 手機版，並且會讓有會員資格的玩家免費體驗遊戲。
我想大家對於GTA這款遊戲都略有耳聞吧？雖然可能不是所有人的家裡都有一台遊戲主機能夠遊玩，但或多或少可能也透過一些影音平台看過相關的遊戲內容。我自己是在大約國中時第一次聽說這款遊戲；然而我手邊既沒有電腦，也沒有遊戲主機，也因此我只能透過 YouTube上的影片來體驗這款遊戲的樂趣。在遊戲開放世界裡，做一些平常做不到（或不能做）的事，擁有現實世界中可能一輩子都買不起的跑車，這些內容都讓那時的我對於GTA有所憧憬。而如今竟然連手機上都可以玩到GTA，儘管可能沒有小時候那樣的渴望，但多多少少對他還是有些好奇跟期待的。
不知道大家怎麼看手機版的GTA呢？會想下載來玩玩看嗎？

**Comment by eric202501** [comment]
我是對其沒有很樂觀，要將這樣的遊戲連接到移動端平台是必要做出許多取捨(畢竟硬體限制)，可能畫質跟遊戲內容會首當其衝。且要讓移動平台兼容需要作出許多調整，不意外應該會有許多bug，畢竟當初三部曲重製剛發售時是挺慘的。而且目前R星應該在收尾GTA6，不太可能放太多心力在這裡。

**Comment by Chengbosco** [comment]
俠盜列車手是很好玩啦，但我不覺得變成手機會好到哪裡去，根據硬體，遊戲肯定會被閹割，甚至會被無聊的家長反彈，說什麼影響小孩(雖然我小時候就在玩盜版了)。與其這樣我還比較期待GTA6。https://www.youtube.com/watch?v=QdBZY2fkU-0

**Comment by davidteng00000** [comment]
手機上推出GTA的消息確實讓人期待，但可能會有一些限制。由於移動平台的硬體限制，可能會導致遊戲畫質及內容上的割捨。再者，GTA6的開發也在進行中，所以投入在手機版上的心力可能會有所減少。雖然對於沒有遊戲主機的玩家是個好消息，但若比起原作的品質差異較大，或許還是期待GTA6會更好。
